src/tools/storage-perf/StoragePerfTool.cpp

```diff
@@ -74,7 +74,10 @@ class Perf {
             threads.emplace_back(std::move(t));
         }
         threadPool_ = std::make_shared<folly::IOThreadPoolExecutor>(FLAGS_io_threads);
-        mClient_ = std::make_unique<meta::MetaClient>(threadPool_, metaAddrsRet.value());
+
+        meta::MetaClientOptions options;
+        options.skipConfig_ = true;
+        mClient_ = std::make_unique<meta::MetaClient>(threadPool_, metaAddrsRet.value(), options);
```

Comment:
Would you please update the code in tools/db-dump/DbDumper.cpp ? It also initiates a meta client.